layers/README.md

```diff
@@ -0,0 +1,85 @@
+| [metadata.py](core/metadata.py) | Implements a basic metadata object. |
+| [technique.py](core/technique.py) | Implements a basic technique object. |
+| [layerops.py](manipulators/layerops.py) | Provides a means by which to combine multiple ATT&CK layer objects in customized ways. A further breakdown can be found in the corresponding section below. |
+
+## layer.py
+Layer.py provides a Layer class, which is the interface used to interact with the rest of the core module, and provides an abstract representation of an ATT&CK layer. The layer class has a collection of methods, the functionality of which is documented in the table below. An example of how to interface with the class to load and retrieve layer objects is documented following that.
+
+| method | description |
+|:-------|:------------|
+| Layer().load_input(_input_) | Loads an ATT&CK layer from either a dictionary or a string representation of a dictionary. |
+| Layer().load_file(_input_) | Loads an ATT&CK layer from a file location specified by the _input_. |
+| Layer().export_file(_input_) | Saves the current state of the loaded ATT&CK layer to a json file denoted by the _input_. |
+| Layer().get_dict() | Returns a representation of the current ATT&CK layer object as a dictionary. | 
+
+#### Example Usage
+
+```python
+example_layer_dict = {
+    "name": "example layer",
+    "version": "3.0",
+    "domain": "mitre-enterprise",
+    "description": "hello, world",
+    ...
+    }
+example_layer_location = "/path/to/layer/file.json"
+example_layer_out_location = "/path/to/new/layer/file.json"
+
+from layers.core.layer import Layer
+
+layer1 = Layer(example_layer_dict)
+layer1.export_file(example_layer_out_location)
+
+layer2 = Layer()
+layer2.load_input(example_layer_dict)
+layer2.get_dict()
+
+layer3 = Layer()
+layer3.load_file(example_layer_location)
+```
+
+## layerops.py
+Layerops.py provides the LayerOps class, which is a way to combine layer files in an automated way, using user defined lambda functions. Each LayerOps instance, when created, ingests the provided lambda functions, and stores them for use. An existing LayerOps class can be used to combine layer files according to the initialized lambda using the process method. The breakdown of this two step process is documented in the table below, while examples of both the list and dictionary modes of operation can be found below.
+
+| method | description |
+|:-------|:------------|
+| LayerOps(score=_score_, comment=_comment_, enabled=_enabled_, colors=_colors_, metadata=_metadata_, name=_name_, desc=_desc_, default_values=_default_values_) | Each of the _inputs_ takes a lambda function that will be used to combine technique object fields matching the parameter. The one exception to this is _default_values_, which is an optional dictionary argument containing default values to provide the lambda functions if elements of the combined layers are missing them. |
```

Comment:
It's a bit confusing that LayerOps constructor is listed as a method in this table. You should probably omit the table and just use subsections instead, e.g "Constructing a layerops" and "processing a layer with a layerops instance".

Also, in your description of default_values, the replace the word "elements" with "techniques".

Reply:
I can break them up into sections.